### PR TITLE
[ADD] account_bank_statement_filter: allow to filter by account code in statement reconcile

### PR DESCRIPTION
**Pull request by agaldona**
Filter account move lines also by account code in bank statement reconcile form.

### CODE REVIEW

**Comment by omar7r** [comment]
:+1:

File: account_bank_statement_filter/README.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+=======================================
+Account Bank Statement Reconcile Filter
+=======================================
+
+This module allows to filter by account code when reconcile bank statements.
```
Comment (pedrobaeza) [inline account_bank_statement_filter/README.rst]: s/reconcile/reconciling

File: account_bank_statement_filter/README.rst [hunk 2]
```diff
@@ -0,0 +1,38 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
```
Comment (pedrobaeza) [inline account_bank_statement_filter/README.rst]: Use latest README template: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst

File: account_bank_statement_filter/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# © 2016 Ainara Galdona - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "Account Bank Statement Filter",
+    "version": "8.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "AvanzOSC",
```
Comment (pedrobaeza) [inline account_bank_statement_filter/__openerp__.py]: Include OCA

File: account_bank_statement_filter/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    "website": "http://www.avanzosc.es",
+    "category": "Custom Module",
```
Comment (pedrobaeza) [inline account_bank_statement_filter/__openerp__.py]: Use a better category

File: account_bank_statement_filter/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    "depends": [
+        "account",
+    ],
+    "data": [
```
Comment (pedrobaeza) [inline account_bank_statement_filter/__openerp__.py]: Remove empty sections

**Comment by agaldona** [comment]
Changes done. I'm not sure if readme file is correct now. @pedrobaeza

**Comment by pedrobaeza** [comment]
Please include a test.

**Comment by agaldona** [comment]
Tests added

**Pull request closed without merging**